File: sources/server/server/Customer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace server
{
    public class Customer
    {
        public String Email { get; set; }
        public String Name { get; set; }
        public String Adr { get; set; }
        public Decimal Tel { get; set; }

        public Customer() { }

        public Customer(String Email, String Name, String Adr, Decimal Tel)
        {
            this.Email = Email;
            this.Name = Name;
            this.Adr = Adr;
            this.Tel = Tel;
        }
    }
}

File: sources/customer/src/com/philippspiess/hotel/api/model/Customer.java
package com.philippspiess.hotel.api.model;

public class Customer {

    public String Email;
    public String Name;
    public String Adr;
    public Long Tel;
    
    
	public String getEmail() {
		return Email;
	}
	public void setEmail(String email) {
		Email = email;
	}
	public String getName() {
		return Name;
	}
	public void setName(String name) {
		Name = name;
	}
	public String getAdr() {
		return Adr;
	}
	public void setAdr(String adr) {
		Adr = adr;
	}
	public Long getTel() {
		return Tel;
	}
	public void setTel(Long tel) {
		Tel = tel;
	}
}
